### PR TITLE
SDPT-663 create deployment service

### PR DESCRIPTION
**Pull request by vangork**
1. add deployment and service to support demo in sdp

### CODE REVIEW

File: charts/ingest-gateway/values.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+pravega:
+  controllerUri: tcp://nautilus-pravega-controller.nautilus-pravega.svc.cluster.local:9090
+
+gateway:
+  authType: secret
+  authEndpoint: http://keycloak.nautilus-system.svc.cluster.local/auth/realms/nautilus/protocol/openid-connect/token
+  image: vangork/ingest-gateway:latest
```
Comment (twincitiesguy) [inline charts/ingest-gateway/values.yaml:7]: we should publish this to an org or corporate account, not a personal account on dockerhub

Comment (vangork) [reply charts/ingest-gateway/values.yaml:7]: does our team have a org account? i can change accordingly.

Comment (twincitiesguy) [reply charts/ingest-gateway/values.yaml:7]: Looks like GitHub Container Registry is free for public images - let's use this if possible

Comment (vangork) [reply charts/ingest-gateway/values.yaml:7]: defer to #5

**Pull request merged**